add-on/chrome/bootstrap.js

```diff
@@ -1226,6 +1228,9 @@ var WindowListener = {
       // remove it from existing tabs (there's no way to do that).
       window.LoopUI.mm.removeDelayedFrameScript(FRAME_SCRIPT);
 
+      let loopMessageManager = window.getGroupMessageManager("loop");
+      loopMessageManager.removeDelayedFrameScript("chrome://browser/content/content.js");
```

Comment:
Yep, I will do that. We won't need the random element for this one though, since the script is part of the browser itself.